.travis.yml

```diff
@@ -4,5 +4,3 @@ before_install:
 language: ruby
 rvm:
  - 2.2
-script:
-  - rake
```

Comment:
This is actually what's run by default, so not necessary. Happy to add it back in if folks like being explicit though.

https://docs.travis-ci.com/user/languages/ruby#Default-Test-Script